src/tree/SubscriptionTreeItem.ts

```diff
@@ -97,6 +99,12 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         executeSteps.push(new VirtualMachineCreateStep());
 
         const title: string = 'Create new virtual machine';
+
+        if (!context.advancedCreation) {
+            // for basic create, default to image Ubuntu 18.04 LTS, the first on the list
+            wizardContext.image = new ImageListStep().getAvailableImages()[0];
```

Comment:
Oof do not like this line

Just make a const like this and reference it both here and inside `getAvailableImages`
```
export const ubuntu1804Image: ImageReferenceWithLabel = {
    label: 'Ubuntu Server 18.04 LTS - Gen1',
    publisher: 'Canonical',
    offer: 'UbuntuServer',
    sku: '18.04-LTS',
    version: 'latest'
}
```

Reply:
Coward!